rcl/CMakeLists.txt

```diff
@@ -41,6 +41,8 @@ export_executable(int32_multinode)
 export_executable(guard_condition)
 export_executable(int32_publisher_subscriber)
 export_executable(ping_pong)
+export_executable(long_sequence_publisher)
```

Comment:
The name could be confused with "a sequence of longs" which is something completely different.
I would use something like "fragmented_publication" and "fragmented_subscription".

Reply:
Done, merge if looks good to you